src/snapshot/index.ts

```diff
@@ -399,10 +399,15 @@ export async function safetySync() {
   })
 
   // Figure out which nodes hold which partitions in the new network
+  const minNodesPerConsensuGroup = 32
```

Comment:
magic number ? why not have parameter in config with static number ?

Reply:
I am a bit confused how this is any different than this in the config. 
```
  sharding: { nodesPerConsensusGroup: 5, nodesPerEdge: 2, executeInOneShard: false },
  ```

Reply:
![image](https://github.com/user-attachments/assets/1a45801e-4284-41fa-95f8-9657e8c43238)

Reply:
What I meant to say was why cant we set `config.sharding.nodesPerConsensusGroup` to a static value like 32 ?
Why is there a need to create a new variable? 
Do we want 2 config variables? 
- One is `config.sharding.nodesPerConsensusGroup` which is dynamic and can change
- the other is `config.sharding.minNodesPerConsensusGroup` which is constant and set to 32. 

We shall then use the max of these 2 values. Does this sound good?

Reply:
I am going to make this change.